doc/release-notes/0.18/0.18.md

```diff
@@ -0,0 +1,172 @@
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/7665">#7665</a></td>
+<td valign="top">Extended platform support for the <tt>-XX:+TransparentHugePage</tt> option</td>
+<td valign="top">OpenJDK8 and later (Linux on POWER and IBM Z)</td>
+<td valign="top">Platform support for the <tt>-XX:+TransparentHugePage</tt> option is now extended to Linux on POWER and Linux on IBM Z. This option takes affect only when Transparent Huge Pages (THP) is set to <tt>madvise</tt> on your system and might increase your application footprint.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/7406">#7406</a></td>
+<td valign="top">New <tt>exit</tt> dump agent and <tt>-XX:[+|-]ExitOnOutOfMemoryError</tt> option</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">The new <tt>exit</tt> dump agent saves information on the state of the VM when the VM shuts down, which can help you diagnose problems. For compatibility, OpenJ9 also supports the HotSpot option <tt>-XX:[+|-]ExitOnOutOfMemoryError</tt>. This option shuts down the VM when a <tt>java.lang.OutOfMemory</tt> error occurs by calling the <tt>exit</tt> dump agent.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/7332">#7332</a></td>
+<td valign="top">LUDCL caching enabled by default</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">This feature was disabled by default in Eclipse OpenJ9 0.17.0 due to issue <a href="https://github.com/eclipse/openj9/issues/7332">#7332</a>. This issue is resolved and the option is reenabled.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/7427">#7427</a></td>
+<td valign="top">Support for Terabyte suffixes for <tt>-X</tt> and <tt>-XX</tt> options</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">OpenJ9 now supports 't' and 'T' suffixes (indicating terabytes) for <tt>-X</tt> and <tt>-XX</tt> options that take a <tt>&lt;size&gt;</tt> parameter.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/7728">#7728</a></td>
+<td valign="top">Extended platform support for pause-less garbage collection</td>
+<td valign="top">OpenJDK8 and later (macOS)</td>
+<td valign="top">Support for pause-less garbage collection (<tt>-Xgc:concurrentScavenge</tt>) is now extended to macOS systems.</td>
+</tr>
+
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/pull/7955">#7955</a></td>
+<td valign="top">Remove restriction on multi-layer shared cache</td>
+<td valign="top">OpenJDK8 and later</td>
+<td valign="top">In the previous release, there is a restriction that the <tt>jvmtiSharedCacheInfo.isCorrupt</tt> field and the  <tt>SharedClassCacheInfo.isCacheCorrupt()</tt> method cannot detect a corrupted cache that has a layer number other than zero. This restriction is now removed.</td>
+</tr>
+
+</table>
+
+
+## Known Issues
+
+The v0.17 release contains the following known issues and limitations:
```

Comment:
I guess this should be `v0.18`?

Reply:
Absolutely, yes ... always easy to miss one in the source :-) I'll update it in my next commit, but will wait for further review comments before pushing. Thanks for spotting!